bin/programs/client/src/main.rs

```diff
@@ -5,21 +5,98 @@
 #![no_std]
 #![cfg_attr(any(target_arch = "mips", target_arch = "riscv64"), no_main)]
 
-use kona_client::{BootInfo, CachingOracle};
-use kona_common::io;
+use alloc::sync::Arc;
+use alloy_consensus::Header;
+use kona_client::{
+    l1::{DerivationDriver, OracleBlobProvider, OracleL1ChainProvider},
+    l2::{OracleL2ChainProvider, StatelessL2BlockExecutor, TrieDBHintWriter},
+    BootInfo, CachingOracle,
+};
 use kona_common_proc::client_entry;
 
 extern crate alloc;
 
 /// The size of the LRU cache in the oracle.
-const ORACLE_LRU_SIZE: usize = 16;
+const ORACLE_LRU_SIZE: usize = 1024;
 
 #[client_entry(0x77359400)]
 fn main() -> Result<()> {
+    #[cfg(feature = "tracing-subscriber")]
+    init_tracing_subscriber(3)?;
```

Comment:
Since we're hardcoding the verbosity level here, let's just inline the tracing initiation and delete `init_tracing_subscriber`?

```suggestion
    let subscriber = tracing_subscriber::fmt().with_max_level(Level::DEBUG).finish();
    tracing::subscriber::set_global_default(subscriber).map_err(|e| anyhow!(e))?;
```